### ISSUE TITLE
Overfitting when training with 2 GPUs

### ISSUE THREAD

**Issue by sosppxo** [yanmin-wu/EDA#2]
Hi,
I tried to train EDA for ScanRefer with 2 A100 GPUs following all your settings, but the performance is about 2% lower than yours. Meanwhile, I obverved the overfitting as the figure shows( after about 21 epochs, the training losses have been decreasing, but verification losses are rising). 
<img width="481" alt="image" src="https://user-images.githubusercontent.com/53434580/224465587-bf2ffc89-42c4-49af-bc07-bc9cf53d166d.png">

**Comment by yanmin-wu** [comment]
Thank you for your interest in our work and your feedback.
+ It seems that you only trained for 51 epochs. We recommend that you consider increasing the training time. 
For your reference, we have included our last two training records below. 
  ```shell
  # 1 [4 * 3090 GPUs]
  epoch 51: 51.87
  epoch 57: 54.05
  epoch 75: 54.22
  # 2 [4 * 3090 GPUs]
  epoch 51: 51.35
  epoch 57: 53.99
  epoch 66: 54.44
  ```
  Please note that slight performance variances of about 1% may occur between our repeated attempts, particularly when using different environments and devices.
+ The trend of the verification loss appears to be normal, which is consistent with our experiments.

**Comment by yanmin-wu** [comment]
Close it since no response. Feel free to re-open it if problems still exist.
